app/src/main/java/com/heig/atmanager/MainActivity.java

```diff
@@ -1,27 +1,55 @@
 package com.heig.atmanager;
 
+import androidx.annotation.NonNull;
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.fragment.app.Fragment;
 import androidx.fragment.app.FragmentTransaction;
 
-import android.content.Intent;
 import android.os.Bundle;
+import android.view.MenuItem;
+import android.view.View;
+
+import com.google.android.material.bottomnavigation.BottomNavigationView;
 
 public class MainActivity extends AppCompatActivity {
 
+    BottomNavigationView dock;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
 
-        loadFragment(new HomeFragment());
-
-        //instantly switches to the Profile activity for testing purposes
-        //Intent myIntent = new Intent(MainActivity.this, ProfileActivity.class);
-        //MainActivity.this.startActivity(myIntent);
+        View dockView = findViewById(R.id.dock_container);
+        dock = dockView.findViewById(R.id.dock);
+        dock.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
```

Comment:
Android doc says 👍 
  | true to display the item as the selected item and false if the item should not be      selected. Consider setting non-selectable items as disabled preemptively to make them appear non-interactive.
-- | --

Should return false, when it's not used